src/chunk.cc

```diff
@@ -566,7 +546,7 @@ Chunk::writeTreeBox()
     }
     if(tree.size() == 1)
     {
-      tree[0][0] = L"─" + tree[0][0];
+      tree[0][0] = (UChar)L'─' + tree[0][0];
```

Comment:
Prefix u and \u escapes exist for this: https://en.cppreference.com/w/cpp/language/string_literal - probably better written as `u'\u2500'`